packages/transaction-controller/src/TransactionController.ts

```diff
@@ -972,6 +983,14 @@ export class TransactionController extends BaseController<
 
     this.onBootCleanup();
     this.#checkForPendingTransactionAndStartPolling();
+    this.registerMessageHandlers();
+  }
+
+  private registerMessageHandlers(): void {
```

Comment:
Minor, we use `#` for private functions so it's private at runtime also.

We also put private methods at the bottom of the file so all the public ones are easier to find.